src/test/java/PointTest.java

```diff
@@ -9,4 +9,31 @@ public void testSimpleAssert(){
         Point p = new Point(10,12);
         assertTrue(p.x == 10 && p.y == 12);
     }
+
+    @Test
+    /**
+     * Tests the Point.isInCircle()-function.
```

Comment:
Not sure this description is enough to qualify as "Each test has a comment (in natural language) summarizing its purpose", though I could be wrong!